torch/csrc/profiler/python/combined_traceback.cpp

```diff
@@ -94,6 +94,7 @@ struct PythonTraceback : public CapturedTraceback::Python {
           py::cast<std::string>(filename),
           py::cast<std::string>(funcname),
           (uint64_t)lineno});
+      Py_DECREF(f_code);
```

Comment:
I mean... simple and wrong? LOL. Code objects can die! We would need some way of telling that the pointer was invalid.

Reply:
PyFrame_GetCode returns a new reference so the code object wont die. There's probably some lifecycle I'm not seeing with the tracebacks that we can use to decref it properly.

Reply:
I don't understand. Previously we transferred ownership of f_code to the captured traceback, and the captured traceback decrefs code when it is done with it. What are you doing now?

Reply:
> and the captured traceback decrefs code when it is done with it. What are you doing now?

Woops sorry I forgot about that. I was thinking instead of decrefing the code object only when the traceback gets destroyed (which doesnt happen because of the circular reference), we decref once we are done using the code object. Forgot to get rid of the destructor decref though.